### PR TITLE
3476 disable serilize deserialize logging

### PR DESCRIPTION
**Pull request by alexsm82**
Removing tracing of value when serializing/deserializing to/from the cache.  Fix for #3476.

### CODE REVIEW

File: dev/com.ibm.ws.session.cache/src/com/ibm/ws/session/store/cache/SerializationInfoCache.java [hunk 1]
```diff
@@ -128,10 +129,12 @@ public byte getIndex() {
             return Long.class;
         }
 
+        @Trivial // reveals customer data
```
Comment (njr-11) [inline dev/com.ibm.ws.session.cache/src/com/ibm/ws/session/store/cache/SerializationInfoCache.java]: Can mark these whole classes @Trivial if the serialize debug includes the type

Comment (alexsm82) [reply dev/com.ibm.ws.session.cache/src/com/ibm/ws/session/store/cache/SerializationInfoCache.java]: Yep, I'll update the whole class as trivial and trace the information in serialize/deserialize

**Comment by alexsm82** [comment]
#build

**Comment by alexsm82** [comment]
#build

**Comment by alexsm82** [comment]
#build

**Comment by alexsm82** [comment]
#build

**Comment by alexsm82** [comment]
#build

**Pull request merged**